api/src/main/java/org/openmrs/module/Module.java

```diff
@@ -530,11 +530,15 @@ public void setExtensionNames(Map<String, String> map) {
 
 	private boolean isNoNeedToExpand() {
 		if (extensionNames == null || extensionNames.isEmpty()) {
-			return true;
+			for(Extension ext : extensions) {
+				if (!extensionNames.get(ext.getPointId()).equals(ext.getClass().getName())) {
```

Comment:
I agree with Andu033. The if statement would be true if extensionNames is null. So trying to access extensionNames while inside the if statement(line 534) would probably lead to a nullPointerException, as is the case with the Travis test fail

Reply:
Thank you for strengthening my point of view.